ras/hwinfo.py

```diff
@@ -87,13 +97,18 @@ def test_version(self):
     def test_help(self):
         self.run_cmd("hwinfo --help")
 
-    def test_debug(self):
+    def test_debug_0(self):
```

Comment:
Can we have function name `test_debug` and drive this as list variable to iterate to that list 

```
for value in [0,1]:
   self.run_cmd("hwinfo --debug %d --disk --log=-" %value)
```

Reply:
Updated test cases as per your suggestion. Added for loop to test the options
test_debug
test_dump

Thanks.